BiG World Fixpack/TheUndying/SETUP-TheUndying.tp2.patch

```diff
@@ -16,6 +16,7 @@
 +    STR_VAR
 +      tra_path      = EVAL ~%MOD_FOLDER%/tra~
 +      reload_array  = tra_reload
++      iconv_path    = ~iconv~  // Big-World-Fixpack extracts the `iconv` folder directly into the game dir

```

Comment:
@Ineth2 iconv must be added into each mod if you want to use HANDLE_CHARSETS

Reply:
Why? Existing Big-World-Fixpack patches for other mods don't do that either, for example:  
[setup-stuffofthemagi.tp2.patch#L20](https://github.com/BiGWorldProject/BiG-World-Fixpack/blob/master/BiG%20World%20Fixpack/stuffofthemagi/setup-stuffofthemagi.tp2.patch#L20)

From past commits (e.g. a8375588fdf48bcfd07985259d7b113abc7a05cb and bdf0a863bd1e302ea1ebfaaf239e7d5ee7c86937), it looks like @agb1 intentionally added the central top-level `iconv` folder precisely so that the executable *doesn't* have to be added into each mod. And that seems sensible to me.

Reply:
Intention of agb1 was good idea but in my opinion patching should be one time operation which should produce fully working code also outside of the game directory. So the patched mods can be ease taken as a source for full mod simply by copying files. Can you follow this new approach for new patch?